### PR TITLE
Update of README file

### PR DESCRIPTION
**Pull request by sittim**
I went through installation of the plug-in and updated the README based on the question I had during the installation process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,21 +2,24 @@ This plugin uses clang for accurately completing C and C++ code.
...
+- Alternatively, you can also put the files in `~/.vim/`
 
 You need Vim 7.3 or higher, compiled with python support and ideally, with
 the conceal feature.
 
 ## Minimum Configuration
 
-- Set the `clang_library_path` variable, example
+- Set the `clang_library_path`, which is the directory containing file named
+  libclang.[dll/so/dylib] or the clang shared library file itself, example:
```
Comment (graywolf) [inline README.md]: I always thought that libclang.so is the shared library file itself...

**Comment by graywolf** [comment]
> Update of README file

heh, not sure if renaming the files falls under "update of ..." category :)

File: README.md (hunk 1 above)
Comment (xaizek) [inline README.md]: I'm also confused by this wording.

Comment (sittim) [inline README.md]: Actually, for me it was confusing to set-up clang, this is from file:

https://github.com/Rip-Rip/clang_complete/blob/master/doc/clang_complete.txt

lines 240-245

```
       				       	*clang_complete-library_path*
       				       	*g:clang_library_path*
If libclang is not in your library search path, set this to the absolute path
where libclang is available. This should either be a directory containing a
file named libclang.[dll/so/dylib] or the clang shared library file itself.
Default: ""
```

So I used the `find` command, to search for all the extensions and only found the .so file on my linux machine.

Comment (xaizek) [inline README.md]: OK, so it means path to directory containing library or path to library file, just bad wording in documentation. Thanks.

**Comment by sittim** [comment]
graywold, it is still README, with a different extension, but what would you call it?

File: README.md (hunk 1 above)
Comment (graywolf) [inline README.md]: tbh I had no idea you can pass the .so file path directly to this setting. things we learn.. :)

**Comment by graywolf** [comment]
Sure, I would just named the PR with more details, that's all. The info about renaming the file is neither in the PR name nor it's description. So I would use something like `Update of readme, renamed to readme.md` or something. But hey, not like my opinion is that important, I'm not the one merging this.

pro tip of the day: if you write `@` in the comment, it's offers you relevant people, helps to prevent typos. Since I'm not native speaker, for the moment I was thinking about if `graywold` is a word I don't know :)

**Comment by xaizek** [comment]
Thanks, merged with some additions on top of it.

**Pull request merged**